src/UI/Activity/Queue/TimeleftCell.js

```diff
@@ -10,24 +10,39 @@ module.exports = NzbDroneCell.extend({
         this.$el.empty();
 
         if (this.cellValue) {
-            if (this.cellValue.get('status').toLowerCase() === 'pending') {
-                var ect = this.cellValue.get('estimatedCompletionTime');
-                var time = '{0} at {1}'.format(FormatHelpers.relativeDate(ect), moment(ect).format(UiSettingsModel.time(true, false)));
+            var status = this.cellValue.get('status').toLowerCase();
+            var ect = this.cellValue.get('estimatedCompletionTime');
+            var time = '{0} at {1}'.format(FormatHelpers.relativeDate(ect), moment(ect).format(UiSettingsModel.time(true, false)));
+
+            if (status === 'pending') {
                 this.$el.html('<div title="Delaying download till {0}">-</div>'.format(time));
-                return this;
-            }
+            } else if (status === 'downloadclientunavailable') {
+                this.$el.html('<div title="Retrying download at {0}">-</div>'.format(time));
+            } else {
+                var timeleft = this.cellValue.get('timeleft');
+                var totalSize = FormatHelpers.bytes(this.cellValue.get('size'), 2);
+                var remainingSize = FormatHelpers.bytes(this.cellValue.get('sizeleft'), 2);
 
-            var timeleft = this.cellValue.get('timeleft');
-            var totalSize = FormatHelpers.bytes(this.cellValue.get('size'), 2);
-            var remainingSize = FormatHelpers.bytes(this.cellValue.get('sizeleft'), 2);
+                if (timeleft === undefined) {
+                    this.$el.html('-');
+                } else {
+                    var duration = moment.duration(timeleft);
+                    var days = duration.get('days');
+                    var hours = FormatHelpers.pad(duration.get('hours'), 2);
+                    var minutes = FormatHelpers.pad(duration.get('minutes'), 2);
+                    var seconds = FormatHelpers.pad(duration.get('seconds'), 2);
 
-            if (timeleft === undefined) {
-                this.$el.html('-');
-            } else {
-                this.$el.html('<span title="{1} / {2}">{0}</span>'.format(timeleft, remainingSize, totalSize));
+                    var formattedTime = '{0}:{1}:{2}'.format(hours, minutes, seconds);
+
+                    if (days > 0) {
+                        formattedTime = days + '.' + formattedTime;
```

Comment:
I don't think moment wouldn't give us much here, good call on the change though.